### PR TITLE
YAML batch_2 update

### PR DESCRIPTION
**Pull request by FedericoRondoni**
E10a1_agricultural_production_productive_area E10a2_agricultural_production_area E10a3_agricultural_production_area_change E10a10_harvesting_evolution E10a5_fruit_production E10a6_harvested_parcels_evolution E10a8_winter_cereals E10a9_agricultural_workers E11_shopping_center_activity

### CODE REVIEW

**Comment by santilland** [comment]
Strange, locally it seems to run fine, might be a connection issue to one of the services, i will try to rerun

**Comment by santilland** [comment]
seems to throw always the same error, searching a bit for potential issues i found:
Windows style newline characters can cause issues in Cygwin
I will try to check the new line format in the file

**Comment by santilland** [comment]
@FedericoRondoni could you push another small change? Is it possible you had a line break in the pull request first comment before?
If yes, maybe your edit fixed it, if not, i can't really think of why it is not working. The build works fine locally.

**Comment by FedericoRondoni** [comment]
@santilland yes, it might be that. How can we solve it?

**Comment by lubojr** [comment]
@FedericoRondoni I'm afraid that after you've edited the PR comment, you need to push a new commit (with some, even tiny changes) to trigger the automatic workflow with the new PR comment applied.

For us rerunning a past workflow would take the PR comment in the original form in the past.

**Comment by FedericoRondoni** [comment]
Hi @santilland and @lubojr now it worked 😊 
Thanks!

**Comment by FedericoRondoni** [comment]
@dmoglioni

File: collections/E10a10_harvesting_evolution.yaml [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+  Spaceborne:
+    Satellite:
+      - Sentinel-1
+    Sensor:
+      - C-SAR
 Agency:
-    - ESA
+  - ESA
+Provider:
+  - Name: Copernicus
```
Comment (aapopescu) [inline collections/E10a10_harvesting_evolution.yaml:20]: actually this indicator is provided by VISTA in a study finded by ESA:
The study is [“Impact of COVID-19 on Harvest of Row Crops”](https://eo4society.esa.int/projects/impact-of-covid-19-on-harvest-of-row-crop/) conducted by [VISTA Remote Sensing in Geosciences GmbH] and funded by the European Space Agency.

File: collections/E10a1_agricultural_production_productive_area.yaml [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+  Spaceborne:
+    Satellite:
+      - Sentinel-2
+    Sensor:
+      - MSI
 Agency:
-    - ESA
+  - ESA
 Provider:
-    - Name: FAO
```
Comment (aapopescu) [inline collections/E10a1_agricultural_production_productive_area.yaml:17]: @FedericoRondoni , this indicator was produced by UCL https://uclouvain.be/en/index.html, please update provider.

File: collections/E10a3_agricultural_production_area_change.yaml [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+  Spaceborne:
+    Satellite:
+      - Sentinel-2
+    Sensor:
+      - MSI
 Agency:
-    - ESA
+  - ESA
 Provider:
-    - Name: FAO
```
Comment (aapopescu) [inline collections/E10a3_agricultural_production_area_change.yaml:17]: @FedericoRondoni provider is UCL, same comment as for E10a2

File: collections/E10a2_agricultural_production_area.yaml [hunk 4]
```diff
@@ -1,23 +1,26 @@
...
+  Spaceborne:
+    Satellite:
+      - Sentinel-2
+    Sensor:
+      - MSI
 Agency:
-    - ESA
+  - ESA
 Provider:
-    - Name: FAO
```
Comment (aapopescu) [inline collections/E10a2_agricultural_production_area.yaml:17]: @FedericoRondoni , provider is UCL, same comment as in E10a1

File: collections/E10a5_fruit_production.yaml [hunk 5]
```diff
@@ -2,20 +2,30 @@ Name: fruit_production
...
+      - Sentinel-2
+      - Pleiades
+    Sensor:
+      - MSI
+      - HiRI
 Agency:
-    - ESA
+  - ESA
+  - CNES
+Provider:
```
Comment (aapopescu) [inline collections/E10a5_fruit_production.yaml:22]: Provider of the dataset is UCL https://uclouvain.be/en/index.html

File: collections/E11_shopping_center_activity.yaml [hunk 6]
```diff
@@ -2,19 +2,27 @@ Name: shopping_center_activity
...
+DataSource:
+  Spaceborne:
+    Satellite:
+      - Sentinel-2
+    Sensor:
+      - MSI
 Agency:
-    - ESA
+  - ESA
+Provider:
```
Comment (aapopescu) [inline collections/E11_shopping_center_activity.yaml:21]: Indicator provided by Sistema https://sistema.at/

**Review by aapopescu — CHANGES_REQUESTED** [review]
@FedericoRondoni please address the comments in the yamls

**Comment by FedericoRondoni** [comment]
> @FedericoRondoni please address the comments in the yamls

I think I have done it. I'm not sure I know how to handle a change request in GitHub, but the files should be good now

**Pull request merged**